packages/loot-core/bin/build-browser.js

```diff
@@ -0,0 +1,78 @@
+#!usr/bin/env node
+const { fsUtil, shellUtil, webpackUtil, Path } = require('../../../bin/utils');
+
+const migrationUtil = require('./migration-util');
+
+const ROOT = Path.join(process.cwd(), '/bin'); //Make path consistent with bash
+const DATA_DIR = Path.join(ROOT, '../../desktop-client/public/data');
+const LOCAL_BUILD_FILES = Path.join(ROOT, '../lib-dist/browser');
+const CLIENT_BUILD_FILES = Path.join(ROOT, '../../desktop-client/public/kcab');
+
+// Add custom webpack args here
+let WEBPACK_ARGS = '';
```

Comment:
This feels a bit messy in JS — would it be possible to restructure the code so this isn’t necessary?